ir_datasets/datasets/trec_fair_2021.py

```diff
@@ -0,0 +1,179 @@
+import json
+import mwparserfromhell
+import codecs
+from typing import NamedTuple, Dict
+import ir_datasets
+from ir_datasets.util import GzipExtract, Cache, Lazy
+from ir_datasets.datasets.base import Dataset, YamlDocumentation, FilteredQueries
+from ir_datasets.formats import BaseQueries, BaseDocs, BaseQrels, TrecQrel
+from ir_datasets.indices import PickleLz4FullStore
+import pyautocorpus
+from itertools import chain
+
+
+_logger = ir_datasets.log.easy()
+
+
+NAME = 'trec_fair_2021'
+
+
+class FairTrecDoc(NamedTuple):
+    doc_id: str
+    title: str
+    text: str
+    markupfreetext: str
+    url: str
+    quality_score: str
+    geographic_locations: str
+    quality_score_disk: str
+
+
+class FairTrecQuery(NamedTuple):
```

Comment:
Can we keep all the query fields? I see there's also:
 - `keywords: List[str]`
 - `scope: str`
 - `homepage: str`
 - (no `rel_docs`, as this is handled by qrels)